app/controllers/api/v1/npq_applications_controller.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+require "csv"
+
+module Api
+  module V1
+    class NPQApplicationsController < Api::ApiController
+      include ApiTokenAuthenticatable
+      include ApiPagination
+
+      def index
+        respond_to do |format|
+          format.json do
+            render json: NPQApplicationSerializer.new(paginate(scope)).serializable_hash
+          end
+
+          format.csv do
+            render body: NPQApplicationCsvSerializer.new(scope).call
+          end
+        end
+      end
+
+    private
+
+      def npq_lead_provider
+        current_api_token.cpd_lead_provider.npq_lead_provider
+      end
+
+      def scope
+        s = npq_lead_provider.npq_profiles.includes(:user, :npq_course)
+        s = s.where("updated_at > ?", Time.iso8601(updated_since_filter)) if updated_since_filter.present?
+        s
+      end
```

Comment:
Full words please, `scope` over `s`

Maybe `applications_scope` or `query_scope`, or `applications` for the method name? We do have `access_scope` below, so just `scope` feels a bit off.

Reply:
I will change method to `query_scope` and local variable to `scope`